### PR TITLE
CON-1360 expired funds

### PR DESCRIPTION
**Pull request by dashton82**

### CODE REVIEW

File: src/SFA.DAS.EmployerFinance.MessageHandlers.UnitTests/CommandHandlers/DraftExpireAccountFundsCommandHandlerTests.cs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                It.Is<IDictionary<CalendarPeriod, decimal>>(c => c.First().Value.Equals(levyOne.FundsIn)), 
+                It.Is<IDictionary<CalendarPeriod, decimal>>(c => c.First().Value.Equals(paymentOne.FundsOut)), 
+                It.Is<IDictionary<CalendarPeriod, decimal>>(c=>c.Count.Equals(0)),
+                expiredFundsPeriod)).Returns(expiredFund);
+            
+            //Act
+            await handler.Handle(message, new TestableMessageHandlerContext(new MessageMapper()));
+
+            //Assert
+            expiredFundsRepository.Verify(x => x.GetDraft(message.AccountId), Times.Once);
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.EmployerFinance.MessageHandlers.UnitTests/CommandHandlers/DraftExpireAccountFundsCommandHandlerTests.cs]: You should probably setup this method and pass the output to GetExpiringFunds then remove the verify...as it is you're not testing that the output is passed correctly.

File: src/SFA.DAS.EmployerFinance.MessageHandlers/CommandHandlers/DraftExpireFundsCommandHandler.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            var accounts = await _accountRepository.GetAllAccounts();
+            var commands = accounts.Select(a => new DraftExpireAccountFundsCommand { AccountId = a.Id, DateTo = message.DateTo});
+
+            var tasks = commands.Select(c =>
+            {
+                var sendOptions = new SendOptions();
+
+                sendOptions.RequireImmediateDispatch();
+                sendOptions.RouteToThisEndpoint();
+                sendOptions.SetMessageId($"{nameof(DraftExpireAccountFundsCommand)}-{now.Year}-{now.Month}-{c.AccountId}");
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.EmployerFinance.MessageHandlers/CommandHandlers/DraftExpireFundsCommandHandler.cs]: Does setting the message id achieve anything?  Other than making it a pain in the arse to rerun?

File: src/SFA.DAS.EmployerFinance.MessageHandlers.UnitTests/CommandHandlers/DraftExpireAccountFundsCommandHandlerTests.cs (hunk 1 above)
Comment (dashton82) [reply src/SFA.DAS.EmployerFinance.MessageHandlers.UnitTests/CommandHandlers/DraftExpireAccountFundsCommandHandlerTests.cs]: Updated

File: src/SFA.DAS.EmployerFinance.MessageHandlers/CommandHandlers/DraftExpireFundsCommandHandler.cs (hunk 2 above)
Comment (dashton82) [reply src/SFA.DAS.EmployerFinance.MessageHandlers/CommandHandlers/DraftExpireFundsCommandHandler.cs]: Removed

**Pull request merged**